tensorflow/lite/kernels/internal/reference/integer_ops/softmax.h

```diff
@@ -57,11 +52,15 @@ inline void Softmax(const SoftmaxParams& params,
   const int depth =
       MatchingDim(input_shape, trailing_dim, output_shape, trailing_dim);
 
-  int32 min_num = std::numeric_limits<T>::min();
-  int32 max_num = std::numeric_limits<T>::max();
-  OutputParams<T> output_params;
+  int32 min_num = std::is_same<InputT, int16_t>::value
```

Comment:
Minimum value for int16 input can still be negative. I think we should separate input min_num from output min_num.

Reply:
int16 input range [-32767, 32768]

this kernel cover two case:
- softmax with int8 input and int8/int16 output.
- softmax with int16 input and int16 output, zero_point=0.

_std::is_same<InputT, int16_t>::value_ is for distinguish these two case

Reply:
int16 input range [-32767, 32768]

this kernel cover two case:
- softmax with int8 input and int8/int16 output.
- softmax with int16 input and int16 output, zero_point=0.

_std::is_same<InputT, int16_t>::value_ is for distinguish these two case